dbt_project.yml

```diff
@@ -1238,6 +1238,9 @@ models:
       op_token:
         optimism:
           +schema: op_token_optimism
+      governance:
+        optimism:
+          +schema: op_governance_optimism
```

Comment:
We may put this schema into config blocks and recover this file.
It may be easier for maintenance in the future.